### PR TITLE
[ISSUE#728]check ChannelHandlerContext whether is null

### PR DESCRIPTION
**Pull request by Hoolefu**
## [ISSUE#728]check ChannelHandlerContext whether is null

check ChannelHandlerContext whether is null

### CODE REVIEW

File: namesrv/src/main/java/org/apache/rocketmq/namesrv/processor/DefaultRequestProcessor.java [hunk 1]
```diff
@@ -383,10 +383,12 @@ private RemotingCommand wipeWritePermOfBroker(ChannelHandlerContext ctx,
 
         int wipeTopicCnt = this.namesrvController.getRouteInfoManager().wipeWritePermOfBrokerByLock(requestHeader.getBrokerName());
 
-        log.info("wipe write perm of broker[{}], client: {}, {}",
-            requestHeader.getBrokerName(),
-            RemotingHelper.parseChannelRemoteAddr(ctx.channel()),
-            wipeTopicCnt);
+        if (ctx != null) {
```
Comment (Hoolefu) [inline namesrv/src/main/java/org/apache/rocketmq/namesrv/processor/DefaultRequestProcessor.java]: check ChannelHandlerContext whether is null

File: namesrv/src/main/java/org/apache/rocketmq/namesrv/processor/DefaultRequestProcessor.java [hunk 2]
```diff
@@ -517,13 +521,6 @@ private RemotingCommand updateConfig(ChannelHandlerContext ctx, RemotingCommand
                 return response;
             }
 
-            if (bodyStr == null) {
```
Comment (Hoolefu) [inline namesrv/src/main/java/org/apache/rocketmq/namesrv/processor/DefaultRequestProcessor.java]: line 512 has init bodyStr

File: namesrv/src/main/java/org/apache/rocketmq/namesrv/processor/DefaultRequestProcessor.java [hunk 3]
```diff
@@ -383,10 +383,12 @@ private RemotingCommand wipeWritePermOfBroker(ChannelHandlerContext ctx,
 
         int wipeTopicCnt = this.namesrvController.getRouteInfoManager().wipeWritePermOfBrokerByLock(requestHeader.getBrokerName());
 
-        log.info("wipe write perm of broker[{}], client: {}, {}",
```
Comment (Hoolefu) [inline namesrv/src/main/java/org/apache/rocketmq/namesrv/processor/DefaultRequestProcessor.java]: check ChannelHandlerContext whether is null

File: namesrv/src/main/java/org/apache/rocketmq/namesrv/processor/DefaultRequestProcessor.java [hunk 4]
```diff
@@ -517,13 +521,6 @@ private RemotingCommand updateConfig(ChannelHandlerContext ctx, RemotingCommand
                 return response;
             }
 
-            if (bodyStr == null) {
-                log.error("updateConfig get null body!");
```
Comment (Hoolefu) [inline namesrv/src/main/java/org/apache/rocketmq/namesrv/processor/DefaultRequestProcessor.java]: check ChannelHandlerContext whether is null

**Comment by ShannonDing** [comment]
link #729

**Comment by RongtongJin** [comment]
LGTM～

**Pull request merged**